src/Common/src/Interop/Unix/libcurl/Interop.SafeCurlHandle.cs

```diff
@@ -30,16 +38,48 @@ public static void DisposeAndClearHandle(ref SafeCurlHandle curlHandle)
 
             protected override bool ReleaseHandle()
             {
-                Interop.libcurl.curl_easy_cleanup(this.handle);
+                libcurl.curl_easy_cleanup(this.handle);
                 return true;
             }
         }
 
         internal sealed class SafeCurlMultiHandle : SafeHandle
         {
+            private bool _pollCancelled = true;
+            private readonly int[] _specialFds = new int[2];
+            private readonly HashSet<int> _fdSet = new HashSet<int>();
+            private int _fdCount = 0;
+            private Timer _timer;
+
+            internal bool PollCancelled
+            {
+                get { return _pollCancelled; }
+                set { _pollCancelled = value; }
+            }
+
+            internal int FdCount
```

Comment:
How does FdCount differ from _fdSet.Count?  I'm not clear on why we need _fdCount.